devtools/conda-envs/test_env.yaml

```diff
@@ -16,6 +16,7 @@ dependencies:
   - tqdm
   - pytorch 1.6
   - torchvision
+  - xgboost
```

Comment:
Don't we need https://anaconda.org/conda-forge/py-xgboost? Also, something to note: I think conda-forge's does not offer GPU support yet. https://github.com/conda-forge/xgboost-feedstock/issues/26